src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -224,7 +235,12 @@
      * @return The referenced message, or null
      */
     @Nullable
-    Message getReferencedMessage();
+    default Message getReferencedMessage()
```

Comment:
Add a reference to getMessageReference in these docs to make it clear what the difference is and why it exists